packages/terra-arrange/src/Arrange.jsx

```diff
@@ -25,21 +25,21 @@ const propTypes = {
    */
   fitEnd: PropTypes.element,
   /**
-   * The vertical orientation of all three containers. It will override the aligment of alignFitStart, alignFill and alignFitEnd if given. One of: `center`, `bottom`, `stretch`.
+   * The vertical orientation of all three containers. It will override the aligment of alignFitStart, alignFill and alignFitEnd if given. One of: `center`, `bottom`, `stretch`, `default`.
    */
-  align: PropTypes.oneOf(['center', 'bottom', 'stretch']),
+  align: PropTypes.oneOf(['center', 'bottom', 'stretch', 'default']),
```

Comment:
Why are we adding a default prop option?

Reply:
Sort of a tester to run by @bjankord. The idea is to have examples for default, center and bottom, and not stretch, but default is stretch.

If the idea is to have top, center, and bottom examples, then you’d probably want default to use something like flex-start, since stretch will fill the height of the container if the consumer doesn’t constrain it.

This is basically just my initial thoughts on making the desired examples, minus stretch.

Reply:
I don't recall the reasoning behind the original implementation around the align prop but I believe that in the dropdown on the doc page, selecting "default", the example would not set the align prop at all.
By not specifying the align prop, the fit containers would have the following "default" styles.
```css
align-self: flex-start;
```
The fill container did not have any align-self styles in this "default" state. What this did was align the content in the fit and fill containers vertically to the top.

I could see adding a new value, `top` that is default value for the align prop that would work similar to how we have today. If teams omit the align prop as they are today, it would default the align prop to `top`. For this PR, I think this change is beyond the scope though.

I can sync up with you on this to keep changes to the component source code to a minimum for this PR.

Reply:
Updated: [68c3a20](https://github.com/cerner/terra-core/pull/2353/commits/68c3a206ca8b941ea1e9f9e7a3e6fad41525c455). I went ahead and removed the default prop and just removed the alignment attributes from all of the examples so they would fallback to the default styling.